unstructured/documents/elements.py

```diff
@@ -381,6 +381,14 @@ def to_dict(self) -> dict:
         return out
 
 
+class Formula(Element):
+    "An element containing formulas in a document"
+
+    category = "Formula"
```

Comment:
Not a blocking comment but just wonder what's an example of extracting formula? Like a math equation? Or something more general?